bcda/api/v1/api.go

```diff
@@ -285,6 +293,36 @@ func ServeData(w http.ResponseWriter, r *http.Request) {
 	} else {
 		http.ServeFile(w, r, fmt.Sprintf("%s/%s/%s", dataDir, jobID, fileName))
 	}
+
+	// Logging request for auditing
+	var jobIdInt, err = strconv.ParseUint(jobID, 10, 64)
+
+	if err == nil {
+		var jobKeys, err = repository.GetJobKeys(context.Background(), uint(jobIdInt))
+
+		if err == nil {
+			var resourceType string
+
+			for _, j := range jobKeys {
+				if strings.TrimSpace(j.FileName) == strings.TrimSpace(fileName) {
+					resourceType = j.ResourceType
+				}
+			}
```

Comment:
This is calling `trimSpace` on `fileName` multiple times.

Is it harmless if `resourceType` isn't set?

Reply:
ah, in the loop, misunderstood, fixed this